.github/workflows/onPushToMain.yml

```diff
@@ -3,12 +3,19 @@ name: version, tag and github release
 
 on:
   push:
-    branches: [main]
+    branches:
+      - main
+      # point at specific branches, or a naming convention via wildcard
+      - prerelease
```

Comment:
Are only wanting this to run on the `prerelease` branch or branches that start with `prerelease`? If it's the latter, this would need to be `prerelease/*`

Reply:
Just prerelease branch.